### DISCUSSION TITLE
Github Education cannot be recertified

### DISCUSSION THREAD

**Discussion by CxMYu** [community/community#143629] [GitHub Education]
### Hi everyone,

A few months ago, I received an email that there was a problem with my certification, that my certification would end early, and that when I chose to re-certify at the end of the day, it was always this interface and I couldn't reapply.
![image](https://github.com/user-attachments/assets/ae8b323e-ab68-4b8b-a0aa-0aca477cf8e4)

**Comment by CxMYu** [comment]
hi🖐

**Comment by lllHunterlllSV** [comment]
**Hello!  @CxMYu 👋**

Thank you for sharing your situation, and I understand how frustrating this must be. Here are a few steps to help you troubleshoot and hopefully resolve the issue with your GitHub Education certification:

- 📧 **Review the Email**: Double-check the email you received to ensure it didn’t contain any specific instructions or a deadline for re-certification. Sometimes, emails about certification issues might include a link or code you need to follow.

- 🔄 **Re-certification Process**: If you’re unable to reapply or re-certify through the interface, it's possible there is a glitch with the page or your account. Try clearing your browser cache or using an **Incognito/Private Mode** window, which can help eliminate any caching issues that might be preventing the process.

- 🧑‍💻 **Certification Status**: Go to your [GitHub Education Dashboard](https://education.github.com/) and check the status of your certification. It might show you whether you can reapply or if any additional steps are needed.

- 🆘 **GitHub Support**: If the issue persists and you’re still unable to reapply or resolve the problem, I recommend reaching out to [GitHub Support](https://support.github.com/). They can look into your account and provide assistance specific to your situation.

I hope these steps help, and that you’re able to resolve the issue quickly. Best of luck, and don’t hesitate to reach out if you need more assistance! 😊
